src/Factories.php

```diff
@@ -26,4 +30,9 @@ public static function all()
 
         return $factories;
     }
+
+    protected static function isLaravel8(): bool
+    {
+        return version_compare(app()->version(), '8.0.0', '>=');
```

Comment:
However 😅 the question still stands regarding Lumen:
- Lumen returns something like `Lumen (8.0.0) (Laravel Components ^8.0)`
- Laravel returns `8.0.3

`version_compare` won't see Lumen >= 8 as `true`. But it shouldn't, no idea 🤷‍♀️

Reply:
Im using this hotfix: 

```
if (class_exists('Illuminate\Database\Eloquent\Factory')) {
//do legacy stuff...
}
```

Reply:
Yeah, maybe that's better / more clear.

I don't use this part so I've a hard time giving useful feedback there 😬